rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/Rename.rsc

```diff
@@ -237,40 +237,42 @@ default list[Tree] extendFocusWithConcreteSyntax(list[Tree] cursor, loc _) = cur
     Augment the TModel with 'missing' use/def information.
     Workaround until the typechecker generates this. https://github.com/usethesource/rascal/issues/2172
 }
-TModel augmentTModel(Tree tr, TModel tm, TModel(loc) tmodelForLoc) {
-    tm = augmentExceptProductions(tr, tm, tmodelForLoc);
-    tm = augmentFieldUses(tr, tm, tmodelForLoc);
-    tm = augmentFormalUses(tr, tm, tmodelForLoc);
-    tm = augmentTypeParams(tr, tm);
+TModel augmentTModel(loc l, TModel tm, PathConfig(loc) getPathConfig) {
+    TModel getModel(loc f) = f.top == l.top ? tm : tmodelForLoc(f, getPathConfig);
+
+    try {
+        tr = parseModuleWithSpaces(l);
+        tm = augmentExceptProductions(tr, tm, getModel);
+        tm = augmentFieldUses(tr, tm, getModel);
+        tm = augmentFormalUses(tr, tm, getModel);
+        tm = augmentTypeParams(tr, tm);
+    } catch _: {;}
```

Comment:
this is very hard to debug/notice a failure. I would recommend at least printing a message somewhere.

Reply:
Of course. Added a print.